### PR TITLE
update project assignments chart for enhancements

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create Project Assignments Chart Screen #274](https://github.com/nerubia/kh360-react/issues/274)

### CODE REVIEW

File: src/features/admin/project-assigments/project-assignments-list.tsx [hunk 1]
```diff
@@ -8,13 +8,28 @@ import { searchProjectMembers } from "../../../redux/slices/project-members-slic
...
+import { type ProjectMember } from "../../../types/project-member-type"
 
 export const ProjectAssignmentsList = () => {
   const appDispatch = useAppDispatch()
   const { project_members } = useAppSelector((state) => state.projectMembers)
 
   const [activeProjectMembers, setActiveProjectMembers] = useState<Task[]>([])
 
+  const uniqueRolesSet = new Set()
+  const legendData = project_members.reduce((result: ProjectMember[], member: ProjectMember) => {
```
Comment (jatnerubia) [inline src/features/admin/project-assigments/project-assignments-list.tsx:20]: can you handle this inside the useEffect

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by chadiegilnerubia — APPROVED** [review]

**Pull request merged**